python/private/hermetic_runtime_repo_setup.bzl

```diff
@@ -109,7 +109,26 @@ def define_hermetic_runtime_toolchain_impl(
     cc_library(
         name = "python_headers",
         deps = select({
-            "@bazel_tools//src/conditions:windows": [":interface", ":abi3_interface"],
+            "@bazel_tools//src/conditions:windows": [":interface"],
+            "//conditions:default": None,
+        }),
+        hdrs = [":includes"],
```

Comment:
Are the headers actually identical for both stable and unstable ABIs? Or, ideally, should stable omit some subset of headers? I'm guessing this is transparently handled by ifdefs, but thought I'd ask for completness/edification.

Reply:
I think so, yes. On that note, I'm not sure if the `{major}.{minor}m` paths actually exist on Windows - I downloaded some Windows assets from `python-build-standalone`, and none of them had any directories other than `include`.

Reply:
Thanks for pointing that out. I field https://github.com/bazel-contrib/rules_python/issues/3281 for the m-suffix being defunct.